examples/ExpoMessaging/android/gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem
```

Comment:
What is the idea behind this change?

Reply:
Automatic generation of `android` file on expo. I didn't change it. Having a .gitignore entry would solve it.